package.json

```diff
@@ -14,15 +14,16 @@
   "main": "index.js",
   "scripts": {
     "test": "lab -cv",
-    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/stylish.js --exclude node_modules"
+    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/index.js --exclude node_modules"
```

Comment:
@phated Just FYI, I had begun switching some of the related repositories to eslint and aligning their .travis.yml as per the above. But just now I noticed this new repo [here](https://github.com/gulpjs/eslint-config-gulp). Looks like I should use that now, correct?